nixops/backends/__init__.py

```diff
@@ -137,6 +145,8 @@ def set_common_state(self, defn) -> None:
         if not self.has_fast_connection:
             self.ssh.enable_compression()
 
+        self.ssh.privilege_escalation_command_set(defn.privilege_escalation_command)
```

Comment:
Should we put `set_` first? What is more pythonic?

Reply:
I don't know what's more pythonic. Personally I tend to like suffixing the operation so editor completion will suggest related operations.

Reply:
The most Pythonic way would actually be _not_ to use setters/getters at all but just mutate the attribute.

Reply:
Let's leave it like this for now and reconsider in the future `SSH`/`SSHMaster` refactor you and me discussed.